test/src.plugins.explorer.spec.js

```diff
@@ -349,6 +349,7 @@ describe('Explorer plugin', function () {
 
   describe('refreshAllTransactions', function () {
     it('should start from birthblock', function (done) {
+      this.timeout(80000)
```

Comment:
this change is because, although the test works, it timeouts. Before this PR, the test was just checking one mocked call, but now tons of them are performed. We should probably revisit this test in the future